modules/migration-about-state-migration.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/about-mtc-3-4.adoc
+// * migration_toolkit_for_containers/about-mtc.adoc
+
+[id="migration-state-migration_{context}"]
+= State migration
+
+You can use {mtc-full} ({mtc-short}) to migrate an application's state.
+
+State migration copies selected persistent volume claims (PVCs) and Kubernetes objects that store an application's state.
```

Comment:
This is the Stage description `Select the persistent volumes you want to migrate and optionally rename the PVC for each on the target cluster.`